homeassistant/components/homepilot/config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Config flow for Rademacher HomePilot integration."""
+
+import logging
+
+import aiohttp
+from pyhomepilot import HomePilotAPI
+from pyhomepilot.auth import (  # pylint:disable=redefined-builtin
+    Auth,
+    AuthError,
+    ConnectionError,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
```

Comment:
drill down the imports to what ever you realy need (_guess we will not need the whole `homeassistant.core` in here_)